Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.3-slim
+MAINTAINER OpenStreetMap
+ENV REFRESHED_AT 2016-09-15
+
+# Install packages
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  ruby-dev \
+  libgdbm-dev \
+  libncurses5-dev \
+  libffi-dev \
+  libyaml-dev \
+  libreadline-dev \
+  libpq-dev \
+  libxml2-dev \
+  imagemagick \
+  libmagickwand-dev \
+  postgresql-client \
+  nodejs \
+  file
+
+# Setup app location
+RUN mkdir -p /app
+WORKDIR /app
+
+# Install gems
+ADD Gemfile /app/Gemfile
+ADD Gemfile.lock /app/Gemfile.lock
+RUN bundle install
```

Comment:
Also consider adding `-j $(nproc)` as an argument to speed up gem installation (it will run a number of processes matching the cores available on the system doing the building).